ci/test/pipeline.template.yml

```diff
@@ -71,10 +71,10 @@ steps:
     label: ":lock: testdrive-ssl"
```

Comment:
Should we change this name to something like `kafka-ssl` or `ssl-smoketest`? And then rename the `kerberos` test to something similar?

Reply:
Ooh yeah, `kafka-ssl` sounds good. I'll send up another commit shortly.